Tasks/AzureResourceManagerTemplateDeploymentV3/operations/FileEncoding.ts

```diff
@@ -0,0 +1,105 @@
+//File Encoding detected to be : utf-32be, which is not supported by Node.js
```

Comment:
this file is lifted as such from previous version. right?

Reply:
Right